src/init/features/dataconnect/sdk.ts

```diff
@@ -75,53 +102,52 @@ async function askQuestions(setup: Setup, config: Config): Promise<SDKInfo> {
     newConnectorYaml.generate = {};
   }
 
-  if (platforms.includes(IOS)) {
-    const outputDir = await promptOnce({
-      message: `What directory do you want to write your Swift SDK code to? (If not absolute, path will be relative to '${connectorInfo.directory}')`,
-      type: "input",
-      default:
-        newConnectorYaml.generate.swiftSdk?.outputDir ||
-        `./../gensdk/${newConnectorYaml.connectorId}/swift-sdk`,
-    });
+  if (targetPlatform === Platform.IOS) {
+    const outputDir =
+      newConnectorYaml.generate.swiftSdk?.outputDir ||
+      path.relative(
+        connectorInfo.directory,
+        path.join(appDir, `generated/${newConnectorYaml.connectorId}`),
+      );
     const pkg = camelCase(newConnectorYaml.connectorId);
```

Comment:
Do we want to check the current `newConnector.generate.swiftSdk.package` value here and default to the connector if there isn't already a value?

Reply:
Good call.